packages/sdk/tests/actions/shutdownGasRelayPaymaster.ts

```diff
@@ -0,0 +1,17 @@
+import { type Address } from "viem";
+import { sendTestTransaction } from "../globals.js";
+import { prepareShutdownGasRelayPaymasterParams } from "../../src/actions/shutdownGasRelayPaymaster.js";
+
+export function shutdownGasRelayPaymaster({
+  account,
+  address,
+}: {
+  account: Address;
+  address: Address;
+}) {
+  return sendTestTransaction({
+    account,
+    address,
+    ...prepareShutdownGasRelayPaymasterParams(),
+  });
+}
```

Comment:
@rayphelan You really don't need to (and maybe also don't want to) create a test action for every small function we have. Instead, re-usable test actions should only be created for things we use a lot across many different tests. Like creating a vault, buying shares, etc.

I see you are using this test action that you created here also in the test cases for the function itself. That has a few downsides. When testing a particular "thing", you usually don't want to test it through some indirection. You are creating artificial indirection here by wrapping the actual thing you want to test in a helper function and then calling it through that instead of directly.

The complexity here is minimal so it's not as big of a deal, but I'm sure you can see how this could be a problem in other cases.

Generally, when testing something, test it directly. 

If you then need that same thing in other tests (that test other stuff), then you can use a utility for convenience there (because there you are not testing that thing itself, but simply need it for e.g. producing some kind of state that is required for the following assertions of the other thing you are testing).